pkg/services/object/server.go

```diff
@@ -138,6 +168,7 @@ func (s *server) Put(gStream protoobject.ObjectService_PutServer) error {
 		if err := stream.Send(putReq); err != nil {
 			if errors.Is(err, util.ErrAbortStream) {
 				resp, err := stream.CloseAndRecv()
+				s.pushOpExecResult(stat.MethodObjectPut, err, t)
```

Comment:
No stat if `!errors.Is(err, util.ErrAbortStream)`?

Reply:
fixed